cpp/Doxyfile.in

```diff
@@ -873,7 +867,11 @@ EXAMPLE_RECURSIVE      = NO
 # that contain images that are to be included in the documentation (see the
 # \image command).
 
-IMAGE_PATH             = @CMAKE_CURRENT_SOURCE_DIR@/doxygen/images
+# IMAGE_PATH             = @CMAKE_CURRENT_SOURCE_DIR@/doxygen/images
```

Comment:
Of course this comment immediately makes me start thinking about what images we can add...